Automattic/pocket-casts-ios
# Use contentType to set extension for downloaded files

Fixes #900

Some podcasts use file URLs with an extension that does not match with the contentType of the files. This changes saves the content file provided by the header response from servers and use that to use as an extension

## To test

- Start the app on the simulator
- Search for the following podcast: `ondacer` on the discovery section
- Subscribe to the podcast
- Open the podcast and download one of the episodes
- Tap play
- The episode should play correctly
- Smoke test other podcasts, to see if this change doesn't brake anything on that front

## Checklist
